flute/csrc/qgemm.cpp

```diff
@@ -7,6 +7,9 @@
 #include "cute/numeric/integral_constant.hpp"
 
 
+at::Tensor fast_hadamard_transform(at::Tensor &x, float scale);
```

Comment:
could we make the signature style similar to others (one line for output type, and one argument per line) --- this is the style used in CUTLASS